docs/reference/treyresearch/README.md

```diff
@@ -41,9 +41,9 @@ By default, all recommendations are enabled. You must explicitly disable them if
   -	A Log Analytics workspace and an Automation account.
   - Azure Sentinel.
   -	A hub virtual network  
-  -	VPN Gateway
-  - ExpressRoute Gateway (optional)
-  -	Azure Firewall
+  -	VPN Gateway (optional - deployment across Availability Zones)
+  - ExpressRoute Gateway (optional - deployment across Availability Zones)
+  -	Azure Firewall (optional - deployment across Availability Zones)
```

Comment:
Can the section about ASC also be updated (free -> _Azure Defender OFF_ and Standard -> _Azure Defender ON_)

Reply:
@hjscherer , I have updated the ASC section.